contracts/token/common/ERC2981/ERC2981Mock.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.8;
+
+import { ERC2981 } from './ERC2981.sol';
+import { ERC2981Storage } from './ERC2981Storage.sol';
+
+import { ERC165 } from '../../../introspection/ERC165.sol';
+
+contract ERC2981Mock is ERC2981, ERC165 {
+    constructor(
+        uint16 royalty,
+        uint16[] memory royalties,
+        address receiver
+    ) {
+        ERC2981Storage.Layout storage l = ERC2981Storage.layout();
+
+        l.royalty = royalty;
+        l.receiver = receiver;
+
```

Comment:
How should storage variables be set? Should each of them have a setter function like `ERC20MetadataStorage` or should they be accessed directly?

Reply:
Access directly.  We are in the process of removing all functions from the storage libraries and replacing them with functions in the internal contracts.